schwifty/iban_registry/generated.json

```diff
@@ -486,16 +486,16 @@
     "iban_length": 27,
     "positions": {
       "account_code": [
-        5,
-        23
+        10,
+        21
```

Comment:
The country specific structure of the IBAN is actually taken from the official [SWIFT IBAN registry](https://www.swift.com/swift-resource/9606/download). Could you maybe elaborate from where you got your information? I'm not from France, and have no intuition if the structure is correct like this or not. It seems however that his is related to this issue https://github.com/mdomke/schwifty/issues/3

Reply:
Now, I understand, your IBAN registry is taken from SWIFT's public IBAN registry which is not accurate.

SWIFT also publishes licensed registries (fairly more extensive), in one of this registries I had access to a few months back, they do specify that positions 5 to 14 in french IBAN and 1 to 10 in french BBAN are actually composed of 2 elements, the bank code and the branch code (code banque & code guichet in french).

There are alternative, more local sources of information providing with the correct specification : 
https://www.ecbs.org/Download/Tr201v3.9.pdf
https://www.ecbs.org/iban/france-bank-account-number.html

IMO, issue is to be adressed with Swift